src/Compilers/VisualBasic/Portable/Binding/Binder_Attributes.vb

```diff
@@ -786,10 +786,10 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                                         Dim cast = DirectCast(node.Syntax, PredefinedCastExpressionSyntax)
 
                                         If cast.Keyword.Kind = SyntaxKind.CObjKeyword Then
-                                            InternalSyntax.Parser.CheckFeatureAvailability(diagBag,
-                                                                                           cast.Keyword.GetLocation(),
-                                                                                           DirectCast(cast.SyntaxTree, VisualBasicSyntaxTree).Options.LanguageVersion,
-                                                                                           InternalSyntax.Feature.CObjInAttributeArguments)
+                                            InternalSyntax.IsAvailable(InternalSyntax.Feature.CObjInAttributeArguments,
```

Comment:
Can the extension syntax be used here? `InternalSyntax.Feature.CObjInAttributeArguments.IsAvailable(...)`

Reply:
Yep it can missed it when moved the extension methods.